### DISCUSSION TITLE
Include pre-release version, only if stable version does not exists yet

### DISCUSSION THREAD

**Discussion by AlexRadch** [NuGet/Home#11193] [Ideas]
Can I include pre-release version only if stable version does not exists yet?

I develop package that in normal should include stable version of other package. For example `6.*`
But now other package does not have stable version. So now I have to include pre-release version `6.*-*`

Later when other package will have stable version I will need to change including to `6.*` in my package to disable using pre-release versions. 

How I can include stable version with optionally included pre-release if stable version does not exist?

**Comment by jbennink** [comment]
You could perhaps add both with a Condition `Condition="'$(Configuration)'=='Release'"` for production ) ie. non-prerelease and Condition="'$(Configuration)'=='Debug'" for development where you do allow pre-release.

O this is very very old :-(
